packages/x-components/src/design-system/components/picture/default.scss

```diff
@@ -1,48 +1,20 @@
 .x-picture {
-  // layout
-  display: block;
-  overflow: hidden;
-
-  // border
-  border-radius: var(--x-size-border-radius-top-picture-default)
-    var(--x-size-border-radius-right-picture-default)
-    var(--x-size-border-radius-bottom-picture-default)
-    var(--x-size-border-radius-left-picture-default);
-
-  // color
-  background: var(--x-color-background-picture-default);
-
   &__image {
-    // layout
-    display: block;
-
     // sizes
     width: 100%;
     height: 100%;
 
-    // others
-    object-fit: var(--x-object-fit-picture-default);
-    mix-blend-mode: var(--x-mix-blend-mode-picture-default);
-
-    &--fallback {
-      --x-object-fit-picture-default: var(--x-object-fit-picture-fallback-default);
-      --x-mix-blend-mode-picture-default: var(--x-mix-blend-mode-picture-fallback-default);
-      & > rect {
-        fill: var(--x-color-fill-picture-fallback-rect-default);
-      }
-      & > path {
-        fill: var(--x-color-fill-picture-fallback-path-default);
-      }
-    }
-
+    &--fallback,
     &--placeholder {
-      --x-object-fit-picture-default: var(--x-object-fit-picture-placeholder-default);
-      --x-mix-blend-mode-picture-default: var(--x-mix-blend-mode-picture-placeholder-default);
+      object-fit: contain;
+      background-color: var(--x-color-base-neutral-95);
+      mix-blend-mode: normal;
+
       & > rect {
-        fill: var(--x-color-fill-picture-placeholder-rect-default);
+        fill: var(--x-color-base-neutral-95);
       }
       & > path {
-        fill: var(--x-color-fill-picture-placeholder-path-default);
+        fill: var(--x-color-base-neutral-70);
       }
```

Comment:
I procrastinated a lot thinking about this....., maybe we could add these styles in x-tailwind?